scripts/weekly-dep-report.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
make sure this file is covered in shellcheck in the ci-test.yml non-rust-lints job.   If using xcode there is a shellcheck plugin you can install to make check your code.   Clean up any violations.

Reply:
to fail fast, and not silently, at the start of the script

# fast fail.
set -eo pipefail